apps/grader-host/src/index.js

```diff
@@ -177,17 +177,12 @@ function isJobCanceled(job, callback) {
 function handleJob(job, done) {
   load.startJob();
 
-  const loggerOptions = {
-    bucket: job.s3Bucket,
-    rootKey: job.s3RootKey,
-  };
-
-  const logger = jobLogger(loggerOptions);
+  const logger = makeJobLogger();
   globalLogger.info(`Logging job ${job.jobId} to S3: ${job.s3Bucket}/${job.s3RootKey}`);
 
   const info = {
     docker: new Docker(),
-    s3: new S3(makeS3ClientConfig()),
+    s3: new S3(makeS3ClientConfig({ maxAttempts: 3 })),
```

Comment:
This should help a bit with resiliency - I've observed some times where we exceed rate limits, though that's probably in part because `s3-streamlogger` would rapidly try to upload log chunks.